src/Tribe/Admin_List.php

```diff
@@ -155,6 +158,41 @@ public static function sort_by_event_date( Array $clauses, WP_Query $wp_query )
 			return $clauses;
 		}
 
+		/**
+		 * Defines custom logic for filtering events table by aggregator record.
+		 *
+		 * @param   Array       $clauses    SQL clauses for fetching posts
+		 * @param   WP_Query    $wp_query   A paginated query for items
+		 *
+		 * @return  Array                   Modified SQL clauses
+		 */
+		public static function filter_by_aggregator_record( Array $clauses, WP_Query $wp_query ) {
```

Comment:
* :no_entry_sign: Expected type hint "array"; found "Array" for $clauses
* :no_entry_sign: PHP parameter type declarations must be lowercase; expected "array" but found "Array"

<sub>hash: 676eac1ffc72db61114f33169c382a71</sub>